ppa/archive/templates/archive/snippets/search_result.html

```diff
@@ -14,10 +14,19 @@
                     {% if page_obj %}
                         <p class="result-number">{{ page_obj.start_index|add:forloop.counter0 }}</p>
                     {% endif %}
+                    {% if item.work_type != "full-work" %}
+                    <div class="work-type icon" aria-label="{{ item.work_type }}"></div>
+                    {% endif %}
                 </div>
                 <p>
+                    {% spaceless %}
                     <span class="ui header">{{ item.title }}</span>
+                    {% if item.work_type != "full-work" and item.book_journal %}
```

Comment:
I think the extra check for type _shouldn't_ be necessary, but right now we don't have much validation on the fields for the different types so I see the value in keeping it